template/src/pages/VideoCall.tsx

```diff
@@ -212,8 +218,10 @@ enum RnEncryptionEnum {
 }
 
 const VideoCall: React.FC = () => {
-  const {store} = useContext(StorageContext);
-  const [username, setUsername] = useState('Getting name...');
+  const {store, setStore} = useContext(StorageContext);
+  const [username, setUsername] = useState(
```

Comment:
use lazily initialise useState for conditional or anything other than a constant value.
const [username, setUsername] = useState( () =>   store?.displayName ? store.displayName : '',);

Reply:
updated